addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
> sure but this happens if the api is crappy, not fine grained enough, and gets changed. this is pre-alpha. there is no better point in time to change this.

crappy or not. it is there. people use it, people used it for years. you can not just bump it because hey it was crappy. and you can not just remove a method, that is part of an api. or change it's signature. you are supposed to "deprecate" for at least one release.

you are dealing not with stupid users here. but developers. (most of) python devs who do kodi addons are not complete idiots. neither skinners are.

> Either we do it right or we don't do it at all. There is no in between. If you continue to call this a stupid change, we will have a hard time, I promise you.

there is in between. see above. this bump is not the right thing. we've had enouh nonsense talk about setMimeType(). and it was broken for two years. you blocked a fix, because the api was "crappy". meh!

> mind your words! you are not even a developer

unlike you, a top of the tops of the devs, who cares about player architecture, and active ae and what not. there are people who have to care about stuff you never do. like addons. yea. he is not always nice as you are. but he also has his reasons.